src/pages/settings/Settings.tsx

```diff
@@ -77,6 +78,17 @@ const Settings: FC = () => {
     type: "string",
   });
 
+  if (hasAccessManagement) {
+    configFields.push({
+      key: "user.show_permissions",
+      category: "user",
+      default: "false",
+      shortdesc:
+        "Show the permissions feature. If oidc configs are set, this setting will not have any effect.",
```

Comment:
```suggestion
        "Show the permissions feature. If oidc configs are set, the permissions feature is available in the UI independent of this setting.",
```
Maybe a bit more clear?